rpms/build/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Container for publishing GeoDocker RPMs to RPM repo on hosted S3
+FROM centos:7
```

Comment:
This probably should be the same Docker image that is used for building RPMs, why not?